apps/core/graphql/types/viewer.ex

```diff
@@ -1,16 +1,52 @@
 defmodule Core.GraphQL.Types.Viewer do
   use Absinthe.Schema.Notation
   use Absinthe.Relay.Schema.Notation
+  alias Absinthe.Relay.Connection
+  alias Core.{GraphQL.Schema, User, Repo}
+  import Ecto.Query, only: [from: 2]
+  import Canada, only: [can?: 2]
+  import Absinthe.Relay.Node, only: [from_global_id: 2]
 
   node object :user do
     field :first_name, :string
     field :last_name, :string
     field :email, :string
-    field :telephone, :string
+    field :phone, :string
+    field :role, :string
+    field :password, :string
+    field :password_confirmation, :string
   end
 
+  @desc """
+  User connection.
+  """
+  connection node_type: :user
+
   object :viewer do
     field :profile, :user
+
+    @desc "El Jurista users"
+    connection field :users, node_type: :user do
+
+      resolve fn
```

Comment:
Please add *authorization* logic here. Only `admin` or `superadmin` `viewer` should be allowed to list users.